### PR TITLE
DEV-2011: Add MyAvatar::isSeated API method

### PR DESCRIPTION
**Pull request by luiscuenca**
https://highfidelity.atlassian.net/browse/DEV-2011
This PR adds a new MyAvatar::isSeated API method. It also adds more information the jsdocs about the MyAvatar.isInSittingState method.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21737/HighFidelity-Beta-PR16298-3a9c829.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21737/HighFidelity-Quest-Beta-PR16298-3a9c829.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21737/HighFidelity-Beta-PR16298-3a9c829.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21737/HighFidelity-Beta-Interface-PR16298-3a9c829.dmg

**Comment by luiscuenca** [comment]
Build this please

File: interface/src/avatar/MyAvatar.h [hunk 1]
```diff
@@ -272,12 +272,12 @@ class MyAvatar : public Avatar {
      *     the value.</p>
      * @property {number} analogPlusSprintSpeed - The sprint (run) speed of your avatar for the "AnalogPlus" control scheme.
      * @property {MyAvatar.SitStandModelType} userRecenterModel - Controls avatar leaning and recentering behavior.
-     * @property {number} isInSittingState - <code>true</code> if your avatar is sitting (avatar leaning is disabled, 
-     *     recenntering is enabled), <code>false</code> if it is standing (avatar leaning is enabled, and avatar recenters if it 
-     *     leans too far). If <code>userRecenterModel == 2</code> (i.e., auto) the property value automatically updates as the 
-     *     user sits or stands, unless <code>isSitStandStateLocked == true</code>. Setting the property value overrides the 
-     *     current siting / standing state, which is updated when the user next sits or stands unless 
-     *     <code>isSitStandStateLocked == true</code>.
+     * @property {number} isInSittingState - <code>true</code> if, while in HMD, your avatar is determined to be sitting
```
Comment (hyperlogic) [inline interface/src/avatar/MyAvatar.h]: Hey, on second read, this should read something like this:

`true` if the user wearing the HMD is determined to be sitting. (avatar leaning is disabled, recenntering is enabled), `false` if the user wearing the HMD is determined to be standing ...

I think "user wearing the HMD" is more accurate then Avatar.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21738/HighFidelity-Beta-PR16298-3a9c829.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21738/HighFidelity-Quest-Beta-PR16298-3a9c829.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21738/HighFidelity-Beta-PR16298-3a9c829.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21738/HighFidelity-Beta-Interface-PR16298-3a9c829.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21740/HighFidelity-Beta-PR16298-07080f0.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21740/HighFidelity-Quest-Beta-PR16298-07080f0.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21740/HighFidelity-Beta-PR16298-07080f0.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21740/HighFidelity-Beta-Interface-PR16298-07080f0.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21740/HighFidelity-Beta-PR16298-07080f0.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21740/HighFidelity-Beta-Interface-PR16298-07080f0.exe

**Pull request merged**